DESCRIPTION

```diff
@@ -21,4 +21,4 @@ License: GPL (>= 2)
 URL: https://github.com/eddelbuettel/drat, https://dirk.eddelbuettel.com/code/drat.html
 BugReports: https://github.com/eddelbuettel/drat/issues
 Encoding: UTF-8
-RoxygenNote: 6.0.1
+RoxygenNote: 7.1.1
```

Comment:
(I use an older/simpler roxygen by default hence the old version)

(As we can see below, a good part of the diff of Rd files is entirely spurious and just line breaks and style of function signature printing)

Reply:
There was one warning for me, where the `location` argument was in the function, but not in the docs.

See line 26 of `man/pruneRepo.rd`

Reply:
Ahh. Good catch.  I may have to (eventually) revert all this back just because the older / faster roxygen is my default here at home.  (The new one needlessly recompiles shared libraries in packages with code which I find annoying but my issue tickets on it were all closed by the powers-that-be.  Oh well.  And, being an ESS user and its indentation, I am not a fan of the new look and "styling".  Ah well squared.  I'll hide under my rock now.)